### PR TITLE
feat(discover): multi y axis support for discover landing page query previews

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
updated `MiniGraph` to accept `yAxis` prop to support previewing multi yaxis on saved queries
image

### CODE REVIEW

**Comment by edwardgou-sentry** [comment]
Removed the `aggregateMultiPlotType` switch from `getChartType` since we don't have a scenario where we would use line charts for Discover at the moment (for multi series and single series).

File: static/app/views/eventsV2/queryList.tsx [hunk 1]
```diff
@@ -244,12 +244,20 @@ class QueryList extends React.Component<Props> {
...
           }}
           renderGraph={() => (
-            <MiniGraph
-              location={location}
-              eventView={eventView}
+            <Feature
               organization={organization}
-              referrer={referrer}
-            />
+              features={['connect-discover-and-dashboards', 'dashboards-edit']}
```
Comment (wmak) [inline static/app/views/eventsV2/queryList.tsx:249]: Out of curiosity why does this feature check need dashboards-edit too?

**Review by wmak — APPROVED** [review]
Question about a feature check but otherwise LGTM

File: static/app/views/eventsV2/queryList.tsx (hunk 1 above)
Comment (edwardgou-sentry) [reply static/app/views/eventsV2/queryList.tsx]: You're right `dashboards-edit` isn't needed here 😄 . Updated!

**Pull request merged**